code/mod_table/mod_table.cpp

```diff
@@ -765,6 +766,10 @@ void parse_mod_table(const char *filename)
 			}
 		}
 
+		if (optional_string("$Use per-weapon impact sounds for hits to player:")) {
```

Comment:
I'd say "$Use weapon class..." in the string too.  "Per-weapon" could imply it can be different for every shot.